tests/integrations/chalice/test_chalice.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+import sentry_sdk
+
+pytest.importorskip("chalice")
+from chalice import Chalice
+
+import sentry_sdk.integrations.chalice as chalice_sentry
+
+from pytest_chalice.handlers import RequestHandler
+
+
+SENTRY_DSN = "https://111@sentry.io/111"
+
+
+@pytest.fixture
+def app():
+    sentry_sdk.init(dsn=SENTRY_DSN, integrations=[chalice_sentry.ChaliceIntegration()])
```

Comment:
Use the `sentry_init` pytest fixture instead, then you dont need to pass a bogus DSN

Reply:
done!